File: 01.JAVA/pg-166-169/pg_166_03.java
import java.util.Scanner;

public class pg_166_03
{
    public static void main(String[] args)
    {
        Scanner enter = new Scanner(System.in);
        int num;

        System.out.println("");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVVJVJ");
        System.out.println("READ AN INTEGER AND DETERMINE IF IT'S NEGATIVE");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJJ");
        System.out.println("");

        System.out.print("Write an integer: ");
        num = enter.nextInt();

        if(num < 0)
        {
            System.out.println("The written number is Negative: " + num);
        }
        else
        {
            System.out.println("The written number isn't negative: " + num);
        }
    }
}

File: 03.JAVASCRIPT/pg-166-169/pg_166_03.js
console.log("");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJS");
console.log("READ AN INTEGER AND DETERMINE IF IT'S NEGATIVE");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJS");
console.log("");

let numInput = document.querySelector("#num");
let button = document.querySelector("#btnProof");
let textar = document.querySelector("#textar");

function validateNumber()
{
    let num = Number(numInput.value);

    if(num < 0)
    {
        textar.value = ("The written number is negative: " + num);
    }
    else
    {
        textar.value = ("The written number ISN'T negative: " + num);
    }

}

button.addEventListener("click", validateNumber);
numInput.addEventListener("keydown", function (event) 
{
    if (event.key === "Enter") {
        validateNumber();
    }

});